common/src/OpenSpaceNet.cpp

```diff
@@ -238,102 +296,81 @@ void OpenSpaceNet::initLocalImage()
 
         bbox_ = intersect;
     }
+
+    GeoBlockSource::Ptr blockSource = GdalBlockSource::create("source");
+    blockSource->attr("path") = args_.image;
+    return blockSource;
 }
 
-void OpenSpaceNet::initMapServiceImage()
+GeoBlockSource::Ptr OpenSpaceNet::initMapServiceImage()
 {
     DG_CHECK(args_.bbox, "Bounding box must be specified");
 
+    std::unique_ptr<deepcore::imagery::MapServiceClient> client;
     bool wmts = true;
     string url;
     switch(args_.source) {
         case Source::MAPS_API:
-            OSN_LOG(info) << "Connecting to MapsAPI..." ;
-            client_ = make_unique<MapBoxClient>(args_.mapId, args_.token);
+            OSN_LOG(info) << "    Connecting to MapsAPI..." ;
```

Comment:
Why did you add spaces here?

Reply:
I had messed around with how some info was output but ended up with much of it exactly the same - reverting this..